src/doc/manifest.md

```diff
@@ -48,6 +48,11 @@ This field specifies a URL to a website hosting the crate's documentation.
 If no URL is specified in the manifest file, [crates.io][cratesio] will
 automatically link your crate to the corresponding [docs.rs][docsrs] page.
 
+Documentation links from specific domainss are blacklisted. Currently, any link
+from rust-ci.org will be blacklisted since this domain is no longer used for
```

Comment:
Could we reorganize this paragraph a bit to make it easier to add more hosts in the future? So instead of having rust-ci.org mentioned in the paragaph, have the paragraph end with: 

> The blacklist currently consists of:
>   - rust-ci.org

so that adding more hosts just entails adding another bullet?